addons/website_sale/static/src/js/website_sale.js

```diff
@@ -163,6 +163,33 @@ publicWidget.registry.WebsiteSale = publicWidget.Widget.extend(VariantMixin, {
 
         this.isWebsite = true;
     },
+    /**
+     * @override
+     */
+    willStart: function () {
+        var self = this;
+        return this._super().then(function (el) {
+            var hash = window.location.hash.substring(1);
+            if (hash) {
+                hash = decodeURIComponent(hash).split(',');
+                var inputs = self.$el.find('input.js_variant_change, select.js_variant_change option');
```

Comment:
- Variables which contain a jQuery element should start with `$`.
- instead of `this.$el.find(`, use the shortcut `this.$(`

Same for below